ProteoformSuiteInternal/ProteoformCommunity.cs

```diff
@@ -83,7 +83,8 @@ public List<ProteoformRelation> relate_ee(Proteoform[] pfs1, Proteoform[] pfs2,
                 from pf1 in pfs1
```

Comment:
I changed line 84 back to just "from pf2 in pfs2" because the where statement was not working correctly for me, things were still being double-counted.